website/project/views/drafts.py

```diff
@@ -121,6 +123,12 @@ def submit_draft_for_review(auth, node, draft, *args, **kwargs):
     :rtype: dict
     :raises: HTTPError if embargo end date is invalid
     """
+    if waffle.switch_is_active(features.OSF_PREREGISTRATION):
+        raise HTTPError(http.GONE, data={
+            'message_short': 'The Prereg Challenge has ended.',
```

Comment:
Nit: `message_short` isn't usually a sentence, because it gets rendered as the page title when the OsfWebRenderer is used. Is this ever rendered to HTML?

Reply:
No this should never actually be hittable by the UI.